ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+### Prerequisites
+
+We thank you for reporting a bug or feature request and helping us to make the SeqAn library better.
+We would like you to check the following points before continuing with the submission as often
+some items might have been solved already. 
+
+* [ ] Are you running the latest version?
+* [ ] Did you check the debugging guide?
+* [ ] Did you check the FAQs on Discuss?
+* [ ] Did you perform a cursory search?
+
+For more information, see the `CONTRIBUTING` guide.
+
+### Description
+
+[Description of the bug or feature]
```

Comment:
I would put this *after* the "expected behaviour", "actual behaviour" and call it "Other notes".